helm/azure-ad-pod-identity-app/templates/nmi-pss-exceptions.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kyverno.io/v2alpha1
```

Comment:
will this thing work on pre 1.25 clusters or will it break helm install?

Reply:
As I understand, this CR was introduced with kyverno 1.9.x and it supports from 1.24 to 1.26.